packages/react-query/src/scripts/utils/package.ts

```diff
@@ -49,3 +51,41 @@ export function getSuspensiveReactQueryPackageJson(targetVersion: string) {
 
   return packageJson
 }
+
+export function getIndexFileContent() {
+  const basePath = path.resolve(__dirname, '../../dist').replace(/src/, '')
+
+  return fs.readFileSync(path.join(basePath, 'index.js'), 'utf-8') || ''
+}
+
+export function getTargetSuspensiveReactQueryVersion(): string {
+  const indexFileContent = getIndexFileContent()
+
+  return (RegExp(/@suspensive\/react-query-(\d+)/).exec(indexFileContent) || [])[1] || ''
+}
+
+export function getTargetSuspensiveReactQueryAPIs() {
+  const indexFileContent = getIndexFileContent()
+
+  const modules = [...indexFileContent.matchAll(/export \* from ['"](.+?)['"]/g)]
+  const results: string[] = []
+
+  for (const module of modules) {
+    const moduleName = module[1]
+    const moduleExports = loadModule<Record<string, unknown>>(moduleName)
+
+    if (moduleExports === undefined) {
+      console.warn('[@suspensive/react-query]', `Module ${moduleName} is not found`)
+      exit(1)
+    }
+
+    const functions = Object.keys(moduleExports)
+      .filter((key) => typeof moduleExports[key] === 'function')
+      .map((fn) => (fn[0] === fn[0].toUpperCase() ? `<${fn}/>` : fn))
+      .reverse()
+
+    results.push(...functions)
```

Comment:
If the name of a function needs to be provided as `<ComponentName/>` by distinguishing whether it is a function component based on its uppercase or not, let's remove the `<ComponentName/>` convention for now. This is because functions that start with an uppercase letter are not always guaranteed to be function components.

If so, I think you can change it like the suggestion code. I believe that ensuring our intent in code is a much more important priority than convention.

```suggestion
    results.push(...Object.keys(moduleExports).reverse())
```

Reply:
> If the name of a function needs to be provided as `<ComponentName/>` by distinguishing whether it is a function component based on its uppercase or not, let's remove the `<ComponentName/>` convention for now. This is because functions that start with an uppercase letter are not always guaranteed to be function components.

I agree. Starting with a capital letter does not guarantee that it is a `Components`.